### PR TITLE
[chore] Upgrade resource names plugin.

### PR DESCRIPTION
**Pull request by lukesneeringer**

### CODE REVIEW

**Comment by hzyi-google** [comment]
What's up with the test?

**Comment by lukesneeringer** [comment]
Okay, we do not need this after all -- it was a problem int he plugin itself.

File: requirements.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ oslo.utils >= 3.41.0, < 4.0.0
 pbr >= 5.4.2, < 6.0.0
 protobuf >= 3.9.0, < 4.0.0
 protobuf3-to-dict==0.1.5
-protoc-java-resource-names-plugin >= 0.0.15, < 0.1.0
+protoc-java-resource-names-plugin >= 0.0.16, < 0.1.0
```
Comment (lukesneeringer) [inline requirements.txt]:
````suggestion
protoc-java-resource-names-plugin >= 0.0.17, < 0.1.0
```

**Comment by lukesneeringer** [comment]
Resurrecting this to bisect between this and #737. If smoke tests pass _here_, we can commit this and know that the gapic-generator hash change is responsible for the extra two files.

**Comment by lukesneeringer** [comment]
Superseded by #737.

**Pull request closed without merging**